.gitattributes

```diff
@@ -0,0 +1,6 @@
+# This file is documented at https://git-scm.com/docs/gitattributes.
+# Linguist-specific attributes are documented at
+# https://github.com/github/linguist.
+
+**/zz_generated.*.go linguist-generated=true
```

Comment:
According to the README at https://github.com/github/linguist, the`=true` part is not needed.

Reply:
oh :sweat_smile: I used the same as on `tektoncd/pipeline` :stuck_out_tongue: